engines/python/setup/djl_python/tests/test_test_model.py

```diff
@@ -53,7 +53,6 @@ def test_all_code(self):
     def test_with_env(self):
         envs = {
             "OPTION_MODEL_ID": "NousResearch/Nous-Hermes-Llama2-13b",
```

Comment:
HF_MODEL_ID

Reply:
HF_MODEL_ID actually clash with OPTION_MODEL_ID. Some configuration doesn't really work

Reply:
Yea ideally should be HF_MODEL_ID, but as Qing said some configurations don't really work with OPTION_MODEL_ID. I'll need to fix some parts of the test here, but will update to HF_MODEL_ID

Reply:
In which case HF_MODEL_ID will clash with OPTION_MODEL_ID?

OPTION_MODEL_ID requires a model directory being defined already, while HF_MODEL_ID is treated as a new model.